.travis.yml

```diff
@@ -8,9 +8,13 @@ env:
 - INTEGRATION=true ES_VERSION=2.4.4 TEST_DEBUG=true
 - INTEGRATION=true ES_VERSION=5.6.9 TEST_DEBUG=true
 - INTEGRATION=true ES_VERSION=6.2.4 TEST_DEBUG=true
+- OSS=true INTEGRATION=true ES_VERSION=6.4.2 TEST_DEBUG=true
```

Comment:
what if we instead only tested against the latest 2.x, 5.x, 6.x and master?
This way we only need to install x-pack on 5.x

Reply:
@jsvd Makes sense - I wasn't sure if there was any value in keeping around the last version of 6.x before x-pack was opened up, but I think you're right that there isn't really.

Reply:
also, taking it a bit further, I see no obvious value of testing `OSS=true`, when it is a subset of OSS=false, thoughts?

Reply:
Yeah, I've still got a little clean up to do here - not quite ready for formal review. Early PR was to get tests running on Travis in case that I might have missed locally.

Reply:
But thanks for the early feedback ;)

Reply:
Ahahaha "go away", got it :D